c2corg_ui/templates/outing/helpers/detailed_outings_attributes.html

```diff
@@ -141,10 +141,9 @@
             <p>
                 % for user in outing['associations']['users']:
                   <%
-                      user_name = user['name'] if 'name' in user else user['username']
                       is_last_participant = loop.last and not outing['locales'][0].get('participants')
                   %>
-                  <a href="#TODO">${user_name}</a>${'' if is_last_participant else ', '}
+                  <a href="#TODO">${user['name']}</a>${'' if is_last_participant else ', '}
```

Comment:
This change has already been handled by another PR and should be neutralized: https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/templates/outing/helpers/view.html#L143-L149